### ISSUE TITLE
unable to quickstart concert

### ISSUE THREAD

**Issue by rayannott** [ADVRHumanoids/concert_description#15]
Hi, Concert team.
I tried to quickstart the Concert as shown in your guide. I followed the preceding steps and installed all the requirements (shown no errors), but the line
`mon launch concert_gazebo concert.launch [rviz:=true]`
just runs silently and I cannot proceed with the setup. What might be the problem here?

Also, I generated the .urdf file using the python script you've provided and set up the config package using MoveIt, but even though
`roslaunch concert_moveit_config demo.launch`
runs successfully and allows me to jump between poses I had created,
neither of the following lets me to open a gazebo world with the robot in it:
`roslaunch gazebo_ros empty_world.launch paused:=true use_sim_time:=false gui:=true throttled:=false recording:=false debug:=true
&&
rosrun gazebo_ros spawn_model -file /home/airat/concert_ws/concert_gazebo.urdf -urdf -x 0 -y 0 -z 1 -model concert`;
`roslaunch concert_moveit_config gazebo.launch`
Thank you!

**Comment by alaurenzi** [comment]
Hi @rayannott ,
do you see the same problem when using the provided docker container ?

**Comment by rayannott** [comment]
> Hi @rayannott , do you see the same problem when using the provided docker container ?

Hello. Yes, I later tried cloning the repo using docker and faced the same problem. However, `man mon` works (which means it is an available command).

**Comment by alaurenzi** [comment]
Could you check if you're able to run plain gazebo from inside the docker container and from your host machine?

Just `gazebo`. Thanks

**Comment by rayannott** [comment]
Just `gazebo` works and launches an empty world. I don't quite get what "running it from inside the docker container" means. Could you please clarify (I'm a bit of a newbie)?

**Comment by rayannott** [comment]
So, I looked into it and it turns out I need to run the `concert_description/docker/run-docker.bash` first and then use the `mon` command.
I installed the nvidia-docker2 and restarted daemon with `sudo systemctl daemon-reload` and `sudo systemctl restart docker` commands as suggested [here](https://github.com/NVIDIA/nvidia-docker/issues/838), but I keep seeing the same error: `docker: Error response from daemon: Unknown runtime specified nvidia.`
Maybe it has something to do with the gpu? As far as I know, my laptop has one (NVIDIA GeForce MX250).
